secant/Features/Send/Views/TransactionConfirmationView.swift

```diff
@@ -0,0 +1,47 @@
+import SwiftUI
+import ComposableArchitecture
+
+struct TransactionConfirmation: View {
+    let viewStore: SendViewStore
+
+    var body: some View {
+        VStack {
+            Text("Send \(String(format: "%.7f", Int64(viewStore.transaction.amount).asHumanReadableZecBalance())) ZEC")
```

Comment:
I recently added this formatter to the SDK's Demo App because just showing a formatted float was cumbersome

````Swift
extension NumberFormatter {
    static let zcashFormatter: NumberFormatter = {
        var formatter = NumberFormatter()
        formatter.maximumFractionDigits = 8
        formatter.maximumIntegerDigits = 8
        return formatter
    }()
}
````

Then when you have to use it at the app level it will look like this

````Swift
      self.balance.text = "\(NumberFormatter.zcashFormatter.string(from: NSNumber(value: totalBalance)) ?? "Nil") ZEC"
````

Still not ideal but better than using `String(format:)`

This is something we will have to handle at SDK level and facilitate to wallet devs